conference/management/commands/create_initial_data_for_dev.py

```diff
@@ -41,8 +40,9 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         conference = get_default_conference()
 
+        today = timezone.now().date()
         ExchangeRate.objects.create(
-            datestamp=date.today(), currency="CHF", rate="1.0"
+            datestamp=today, currency="CHF", rate="1.0"
```

Comment:
Should the currency be a parameter we pass in the arguments of the command? And maybe have `EUR` as default?

Reply:
Looking at how the exchange rates are used, EUR is the base currency, so we don't ever convert from EUR. Looks like we only ever use exchange rates when generating invoices, and even last year we've only put the EUR values on invoices - chances are, we might not really need exchange rates anymore (but that's out of the scope of this MR)

Reply:
I get it. Anyway, I only now realized this is for dev, so no biggie.